system/Database/SQLite3/Connection.php

```diff
@@ -265,9 +265,19 @@ protected function _fieldData(string $table): array
      */
     protected function _indexData(string $table): array
     {
-        // Get indexes
-        // Don't use PRAGMA index_list, so we can preserve index order
-        $sql = "SELECT name FROM sqlite_master WHERE type='index' AND tbl_name=" . $this->escape(strtolower($table));
+        $sql = "SELECT 'PRIMARY' as indexname, l.name as fieldname, 'PRIMARY' as indextype FROM pragma_table_info('" . trim($table, '`') . "') as l WHERE l.pk <> 0 UNION ALL
```

Comment:
Why do we need to ```trim($table, '`')```?

Reply:
Aren't the back-ticks desired here? I just did a bunch of work with tables in SQLite3 and used them everywhere... maybe this function is some special exception?

Reply:
Ah or maybe this is getting re-escaped below so the back-ticks were doubling?